test/unit_tests/test_KernelFunctions_correctness.jl

```diff
@@ -39,3 +48,33 @@ A1 = π * r1^2
 A2 = π * r2^2
 @test kernel(x, y) == Ec * (r1 + r2)^2 * abs(A1 - A2)
 @test kernel(x, y) == kernel(y, x)
+
```

Comment:
We should add opt tests as well for `get_normalized_kernel_func` and the `LongKernel`

Reply:
They are added